lib/commands/cache.js

```diff
@@ -0,0 +1,8 @@
+const { getBuildDir } = require('../utils/builds');
+
+async function cache({ cwd, host, user, repo }) {
+  const cacheDir = await getBuildDir(cwd, host, user, repo);
+  console.log(cacheDir);
```

Comment:
We should also return `cacheDir`

Reply:
It's a command, it doesn't return.
The util already returns

Reply:
we exposing the commands in index file. So, a tool can we dependent on build-stats and get data,:

```js
const {cache} = require('build-stats');
await cache(args) // this should give me the path to cache.
```